### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by shuaiLiWang**
## What does this implement/fix? Explain your changes.
1.Fix the bug of unsuccessful creation of eth wallet through keystore import
2.Fix eth address verification failure
3.Support for choosing a coin type when creating an HD wallet
4.Unified BTC/ETH network settings configuration
5.Wallet list support for filtering hardware wallets

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
https://github.com/OneKeyHQ/TaskHub/issues/642
https://github.com/OneKeyHQ/TaskHub/issues/669#event-4220772226
https://github.com/OneKeyHQ/TaskHub/issues/670#issuecomment-762637147

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
App for Android
## Any other comments?
…

### CODE REVIEW

File: electrum_gui/android/console.py [hunk 1]
```diff
@@ -182,6 +182,10 @@ def __init__(self, android_id=None, config=None, user_dir=None, callback=None, c
         # its callback before the daemon threads start.
         self.daemon = daemon.Daemon(self.config, fd)
         self.coins = read_json('eth_servers.json', {})
+        if constants.net.NET == "Bitcoin":
```
Comment (huazhouwang) [inline electrum_gui/android/console.py:185]: 在最新的代码里，chain_type 是由外部指定的，但这里会覆写 chain_type 的值

File: electrum_gui/android/console.py [hunk 2]
```diff
@@ -2724,16 +2729,19 @@ def create_hd_wallet(self, password, seed=None, passphrase="", purpose=84, stren
             print("seed type = %s" % type(seed))
 
         ##create BTC-1
-        wallet_info = self.create("BTC-1", password, seed=seed, passphrase=passphrase,
+        create_coin_list = json.loads(create_coin)
+        if ("btc" in create_coin_list):
```
Comment (huazhouwang) [inline electrum_gui/android/console.py:2733]: 这里的 `()` 可以不用加，加了就生成个 tuple 了

File: electrum_gui/android/sort_wallet_list.py [hunk 3]
```diff
@@ -23,7 +23,7 @@ def sort_wallet_by_hd(self, wallets_info):
         hd_wallets = []
 
         for key, info in wallets_info.items():
-            if -1 != info['type'].find('derived'):
+            if -1 != info['type'].find('derived') and -1 == info['type'].find('-hw-'):
```
Comment (huazhouwang) [inline electrum_gui/android/sort_wallet_list.py:26]: 建议用 `'derived' in info['type'] and '-hw-' in info['type']` 更加易读

File: electrum_gui/android/console.py (hunk 1 above)
Comment (shuaiLiWang) [reply electrum_gui/android/console.py:185]: 这问题是因为 我想了一下 还是直接用BTC的网络判断好了，这样 app就不用再改任何东西了

Comment (huazhouwang) [reply electrum_gui/android/console.py:185]: 我感觉暴露 chain_type 给外部也有利于开发在本地调试

Comment (shuaiLiWang) [reply electrum_gui/android/console.py:185]: 开发调试的时候 肯定是BTC和ETH同步的 调试测试网 那BTC和ETH全都选择测试网，主网同理，我唯一想到暴露出来的目的就是 当BTC和ETH不同步的时候才会用到

Comment (huazhouwang) [reply electrum_gui/android/console.py:185]: ack

**Review by huazhouwang — APPROVED** [review]

**Pull request merged**